controllers/CommentController.js

```diff
@@ -0,0 +1,66 @@
+const Comment = require("../models/Comment");
+const { ObjectId } = require("mongoose");
+
+const createComment = async (req, res) => {
+    console.log(req.body);
+    try {
+      const comment = await Comment.create(req.body);
+      res.status(201).send(comment);
```

Comment:
change to 200

Reply:
`201` indicates that operation was successful and resource(s) has been created
`200` indicates that operation was successful only
so I think `201` is used correctly